src/Interfaces/review/Review.ts

```diff
@@ -0,0 +1,9 @@
+export default interface Review {
+    id: string | null;
+    rating: number | null;
+    createdAt?: Date | string;
+    updatedAt?: Date | string;
+    // has one
+    userId?: string | null;
+    productVariantId?: string | null;
```

Comment:
I wrote this down wrong on the ticket - should be relationship properties:

```suggestion
    user?: User;
    productVariant?: ProductVariant;
```